.env

```diff
@@ -1,6 +1,7 @@
-REACT_APP_API_URL="http://192.168.109.145:8080"
+REACT_APP_API_URL="http://localhost:8080"
 
-REACT_APP_CONTRACT_ADDRESS="0xDd426058553fF2811731Ad2eDd25216f46c982eE"
+REACT_APP_ARTTOKEN_CONTRACT_ADDRESS="0x6823bfC7684C785C9f81C84bDdF5407c36889378"
```

Comment:
`ARTTOKEN_CONTRACT_ADDRESS` shouldn't be here.
It is variable which needs to be read from the `TOKEN MANAGER CONTRACT`.